Library/Formula/nuxeo.rb

```diff
@@ -0,0 +1,56 @@
+class Nuxeo < Formula
+  desc "Enterprise Content Management"
+  homepage "https://nuxeo.github.io/"
+  url "http://cdn.nuxeo.com/nuxeo-7.10/nuxeo-cap-7.10-tomcat.zip"
+  version "7.10"
+  sha256 "9e95fb4e3dd0fdac49395ae8b429c6cc7db3c621ecd4d0dff74ea706a2aba723"
+
+  depends_on "poppler" => :recommended
+  depends_on "pdftohtml" => :optional
+  depends_on "imagemagick"
+  depends_on "ghostscript"
+  depends_on "ufraw"
+  depends_on "libwpd"
+  depends_on "exiftool"
+  depends_on "ffmpeg" => :optional
+
+  def install
+    libexec.install Dir["#{buildpath}/*"]
+
+    bin.install_symlink "#{libexec}/bin/nuxeoctl"
+
+    # TODO: https://jira.nuxeo.com/browse/NXBT-966 Use a wrapper
+    inreplace "#{libexec}/bin/nuxeoctl" do |s|
+      # See https://jira.nuxeo.com/browse/NXP-18418 nuxeoctl symlink is not correctly resolved and breaks NUXEO_HOME resolution.
+      s.gsub! /\${NUXEO_HOME:=.*$/, '${NUXEO_HOME:=$(cd "$(dirname `PRG=$0; while [ -h "$PRG" ] ;do ls=\\\\`ls -ld "$PRG"\\\\`; link=\\\\`expr "$ls" : ".*-> \\\\(.*\)\$"\\\\`; if expr "$link" : "/.*" &> /dev/null; then PRG="$link"; else PRG="\\\\`dirname "$PRG"\\\\`/$link"; fi; done; echo $PRG`)"/..; pwd -P)}'
+      s.gsub! /\${NUXEO_CONF:=.*$/, "${NUXEO_CONF:=\"#{etc}/nuxeo.conf\"}"
+    end
+
+    inreplace "#{libexec}/bin/nuxeo.conf" do |s|
+      s.gsub! /#nuxeo\.log\.dir.*/, "nuxeo.log.dir=#{var}/log/nuxeo"
```

Comment:
You probably want to `.mkpath` these elsewhere in `def install` so you know they'll definitely exist?